portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/MyFormTests.testcase

```diff
@@ -0,0 +1,112 @@
+@component-name = "portal-web-forms-and-data-lists"
+definition {
+
+	property testray.main.component.name = "Forms";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		Navigator.openURL();
+	}
+
+	@description = "validate entries in english and portuguese"
+	@priority = 5
+	test CheckIsPossibleSubmitFormWithTwoDifferentLanguages {
+		task ("Add a form and an edited text field to full name, and back to another field") {
+			FormsAdmin.addForm();
+
+			Form.editName(formName = "New Forms");
+
+			Form.gotoAddField(
+				fieldPositionNumber = 0,
+				fieldType = "Text");
+
+			Form.editFieldLabelMetal(fieldValue = "Full Name");
+
+			Form.closeSidebar();
+		}
+
+		task ("Add a numerical field and edit it for age") {
+			Form.gotoAddField(
+				fieldPositionNumber = 1,
+				fieldType = "Numeric");
+
+			Form.editFieldLabelMetal(fieldValue = "Age");
+		}
+
+		task ("Adds a new translation for the form (portuguese) and edits full name to full name and age to age") {
+			Form.addTranslation(localizedLanguage = "Portuguese (Brazil)");
+
+			FormViewBuilder.editFieldLabel(
+				fieldLabel = "Full Name",
+				label = "Nome Completo");
+
+			FormViewBuilder.editFieldLabel(
+				fieldLabel = "Age",
+				label = "Idade");
+		}
+
+		task ("We publish the form, save the form, and navigate to the published form") {
+			Form.publishForm();
+
+			Form.save();
+
+			FormsAdminNavigator.gotoPublishedForm();
+		}
+
+		task ("Aentering values into the form and sending the entered values") {
+			FormViewBuilder.editText(
+				fieldName = "Full Name",
+				fieldValue = "Henrique");
+
+			FormViewBuilder.editText(
+				fieldName = "Age",
+				fieldValue = 23);
+
+			FormPortlet.submit();
+
+			Form.submitAgain();
+		}
+
+		task ("Changing the language to portuguese and inserting new values and sending") {
+			FormViewBuilder.changeAppLanguageTo(changeAppLanguageTo = "en-US");
+
+			FormViewBuilder.changeAppLanguageTo(changeAppLanguageTo = "português-Brasil");
+
+			FormViewBuilder.editText(
+				fieldName = "Nome Completo",
+				fieldValue = "Jose Henrique Santos de lima");
+
+			FormViewBuilder.editText(
+				fieldName = "Idade",
+				fieldValue = 23);
+
+			FormPortlet.submit();
+
+			Form.submitAgain();
```

Comment:
Since the next step is navigate to the portal, I dont think this step is necessary, can you try remove it and try again ? Thanks

Reply:
:yellow_circle: Discard this, tested locally, it's needed.